Do not update the mime database when installing the port, we do it ourselves
later.
--- skrooge/CMakeLists.txt~	2012-08-20 17:09:15.000000000 -0300
+++ skrooge/CMakeLists.txt	2012-08-20 17:09:26.000000000 -0300
@@ -35,5 +35,4 @@
 find_package(SharedMimeInfo REQUIRED)
 
 install(FILES x-skg.xml DESTINATION ${XDG_MIME_INSTALL_DIR})
-update_xdg_mimetypes(${XDG_MIME_INSTALL_DIR})
 
